### ISSUE TITLE
OSX: first right click menu doesn't allow selecting options

### ISSUE THREAD

**Issue by virtuald** [exaile/exaile#330]
It's even worse on the playlist column menu

**Comment by genodeftest** [comment]
> It's even worse on the playlist column menu
Is this related to #201?

Anyway, I've noted that in many places the code uses `button-` and `key-` events where it should use events like `activate` and `popup-menu`.

**Comment by virtuald** [comment]
I don't think they're related (though, it would be good to migrate to use the correct events where possible). My suspicion is the new message I'm getting ALL THE TIME on Fedora 25:

```
(exaile.py:21535): Gdk-CRITICAL **: xdg_popup_configure: assertion 'impl->transient_for' failed
```

This message goes away when I call `menu.attach_to_widget(widget, None)`. I'll have to try it on OSX this weekend and see what happens.

**Comment by virtuald** [comment]
Confirmed that the fixes applied in b47fc1eff04b792b837fbacec5ed62b3e3a94007 fixed this issue on OSX. There are a lot of menus that aren't attached to something in Exaile though... :(

**Comment by genodeftest** [comment]
> There are a lot of menus that aren't attached to something in Exaile though... :(

I've opened #347 so we do not forget about stuff like this.

> Confirmed that the fixes applied in b47fc1e fixed this issue on OSX. 

Closing as per this comment.
